activity/activity.info

```diff
@@ -1,7 +1,7 @@
 [Activity]
 name = JAMClock
 activity_version = 2
-service_name = org.laptop.JAMClock
+bundle_id = org.sugarlabs.JAMClock
```

Comment:
Why change the name from org.laptop to org.sugarlabs?  This will break resuming any journal entries after the activity is resumed.